doc/parameters.md

```diff
@@ -28,6 +28,29 @@ Specifies the period (in seconds) that the configuration will be stored in the c
 
 This parameter is also used to store OpenID discovery configuration in the local cache, as the same behavior as described above.
 
+### `APICAST_CONFIGURATION_CACHE_STALE`
+
+**Values:** _a positive number_
+**Default:** 0
+
+You can use this variable to specify the number of times that gather
+configuration can consequently fail. This is useful when any external issue on
+reload Apicast configuration, so APICast will use stale information, and it'll
+not delete the caching configuration. 
+
+Note, if a service is deleted, and this value is set, the service will be
+present in APICast configuration for the number of times that this parameter
+set.
+
+The number means, that `APICAST_CONFIGURATION_CACHE` time will be multiplied by
```

Comment:
I'm not sure this is the best option from a UX perspective. Wouldn't it be clearer if this ENV just indicated the number of seconds directly instead of being a multiplier of another ENV?

Reply:
I'm not sure, because time can be lower than the APICAST_CACHE, if someone forgot to change that the values will be a bit mess. 

With this way, the value is always CACHE time plus one stale configuration.